File: Examples/CSharp/Programming-Documents/Comments/AddComments.cs
using System;
using System.Collections;
using System.IO;

using Aspose.Words;

namespace Aspose.Words.Examples.CSharp.Programming_Documents.Comments
{
    class AddComments
    {
        public static void Run()
        {
            //ExStart:AddComments
            // The path to the documents directory.
            string dataDir = RunExamples.GetDataDir_WorkingWithComments();
            //ExStart:CreateSimpleDocumentUsingDocumentBuilder
            Document doc = new Document();
            DocumentBuilder builder = new DocumentBuilder(doc);
            builder.Write("Some text is added.");
            //ExEnd:CreateSimpleDocumentUsingDocumentBuilder
            Comment comment = new Comment(doc, "Awais Hafeez", "AH", DateTime.Today);
            builder.CurrentParagraph.AppendChild(comment);
            comment.Paragraphs.Add(new Paragraph(doc));
            comment.FirstParagraph.Runs.Add(new Run(doc, "Comment text."));
           
            dataDir = dataDir + "Comments_out_.doc";
            // Save the document.
            doc.Save(dataDir);
            //ExEnd:AddComments
            Console.WriteLine("\nComments added successfully.\nFile saved at " + dataDir);
        }        
    }
}

File: Examples/VisualBasic/Programming-Documents/Comments/AddComments.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections
Imports System.IO
Imports System.Reflection
Imports Aspose.Words
Public Class AddComments
    Public Shared Sub Run()
        ' ExStart:AddComments
        ' The path to the documents directory.
        Dim dataDir As String = RunExamples.GetDataDir_WorkingWithComments()
        ' ExStart:CreateSimpleDocumentUsingDocumentBuilder
        Dim doc As New Document()
        Dim builder As New DocumentBuilder(doc)
        builder.Write("Some text is added.")
        ' ExEnd:CreateSimpleDocumentUsingDocumentBuilder
        Dim comment As New Comment(doc, "Awais Hafeez", "AH", DateTime.Today)
        builder.CurrentParagraph.AppendChild(comment)
        comment.Paragraphs.Add(New Paragraph(doc))
        comment.FirstParagraph.Runs.Add(New Run(doc, "Comment text."))

        dataDir = dataDir & Convert.ToString("Comments_out.doc")
        ' Save the document.
        doc.Save(dataDir)
        ' ExEnd:AddComments
        Console.WriteLine(Convert.ToString(vbLf & "Comments added successfully." & vbLf & "File saved at ") & dataDir)
    End Sub
End Class
